src/setup-transport.coffee

```diff
@@ -0,0 +1,21 @@
+colors = require './colors'
+prefixes = require './prefixes'
+{ getGlobalScope, isBrowser } = require './utils'
+
+setupTransport = (transport) ->
+  globalScope = getGlobalScope()
+  browser = isBrowser(globalScope)
+
+  if browser
+    transport.setLogLevelPrefixes(prefixes.browser)
+    transport.setLogLevelStyles(colors.browser)
+  else
+    transport.setLogLevelPrefixes(prefixes.cli)
+    transport.setLogLevelColors(colors.cli)
+  
+  return transport
```

Comment:
It does not make sense to me why the same transport instance should be returned.